metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
Should these licensing issues hold up this PR?  It's not clear to me that the potential problems we've found so far are even issues because they are transitive dependencies of libraries used for testing and/or tooling and are not part of the source or shipped with any binaries.  I would like to point out that our verify_licenses.sh script excludes Maven dependencies with scope set to either "test", "runtime" or "provided".  If these are an issue then we could potentially have issues on the platform side as well.

Also, the new dependencies that were introduced in this PR are both MIT licensed and compatible.  The dependencies in question are already in master.